graphicsmagick/src/utils/init.rs

```diff
@@ -1,21 +1,12 @@
 use graphicsmagick_sys::InitializeMagick;
-use std::{ptr::null, sync::Once, thread};
+use std::{ptr::null, sync::Once};
 
 static HAS_INITIALIZED: Once = Once::new();
 
 /// Wrapper of `graphicsmagick_sys::InitializeMagick`, call it before any `graphicsmagick` action.
-/// Must be call in the main thread.
 pub fn initialize() {
-    HAS_INITIALIZED.call_once(|| {
-        assert_eq!(
-            thread::current().name(),
-            Some("main"),
-            "You have to call `graphicsmagick::initialize` in main thread"
-        );
-
-        unsafe {
-            InitializeMagick(null());
-        }
+    HAS_INITIALIZED.call_once(|| unsafe {
+        InitializeMagick(null());
```

Comment:
@jmjoy I've removed the constraint of having to call this in main thread because the official doc only requires `InitializeMagick` to be called before any other APIs.

I've removed this in this PR because that's what cause the CI to fail:

The testing is performed in non-main thread and thus the `initialize` function will always fail.

Reply:
OK.